src/data/animations/battle-anims.ts

```diff
@@ -449,35 +512,40 @@ export abstract class BattleAnim {
 
     for (const frame of frames) {
       let { x, y } = frame;
-      const scaleX = (frame.zoomX / 100) * (!frame.mirror ? 1 : -1);
-      const scaleY = frame.zoomY / 100;
+      const zoomX = (frame.zoomX / 100) * (!frame.mirror ? 1 : -1);
+      const zoomY = frame.zoomY / 100;
```

Comment:
NIT: Destruct from frame where possible:
```suggestion
      const { mirror } = frame;
      const zoomX = (frame.zoomX / 100) * (!mirror ? 1 : -1);
      const zoomY = frame.zoomY / 100;
```

(Could also extend the existing `let { x, y } = frame;` but then your `mirror` variable is modifiable)